README.md

```diff
@@ -1,57 +1,2567 @@
-# Install and Run
```

Comment:
I know this is a create-react-app addition/modification, but I'm thinking we should probably pare this down to only what's relevant to the app (unless you want to argue it's all technically relevant...which I can see, but it still seems like a bit much).